FAQ: add section for project deletion (#90)
radicle-dev/radicle-docs: docs/understanding-radicle/faq.md

If you are still running into problems, please submit a request in our [#support
channel][sc].

## Can I delete a project?

> Currently this feature is not supported, it will come in one of the upcoming
> releases. After it landed in the clients, it still will not be possible to
> remove projects from the network.

Reviewer: ```suggestion
Currently, this feature is not supported but it is on the roadmap and will be included in an upcoming
release.
```

Reviewer: Confused with what you mean by "After it landed in the clients" — do you mean even when the feature is included?

Author: Exactly, when it will be supported in the client(s) (Upstream, potentiall CLI) that doesn't mean it will be deleted from peers that have replicated it from you. This will never be possible as peers have agency what to do with their local data.

I know I didn't word this elegantly, rephrase how ever you see fit.

Reviewer: Gotcha.

_Currently, this feature is not supported but it is on the roadmap and will be included in an upcoming
release. With this feature, you can only remove your project from your local machine and any connected seed nodes, thus limiting the number of peers who can find and replicate your project. You can not delete a project from another peer's local machine, as they retain control over their local data._ ?

Author: Looks good, except the seed node part. :> Will adopt.

If you are still running into problems, please submit a request in our [#support
channel][sc].

## Can I delete a project?
Currently, this feature is not supported but it is on the roadmap and will be

> included in an upcoming release. With this feature, you can only remove your

Reviewer 2: ```suggestion
included in an upcoming release. Until then, you can only remove your
```
